infra/base-images/base-runner/Dockerfile

```diff
@@ -105,6 +105,11 @@ RUN wget https://repo1.maven.org/maven2/org/jacoco/org.jacoco.cli/0.8.7/org.jaco
     echo "37df187b76888101ecd745282e9cd1ad4ea508d6  /opt/jacoco-agent.jar" | shasum --check && \
     echo "c1814e7bba5fd8786224b09b43c84fd6156db690  /opt/jacoco-cli.jar" | shasum --check
 
+ENV PERF_DIR=/root/perf
+RUN git clone --depth 1 https://github.com/brendangregg/FlameGraph $PERF_DIR
```

Comment:
Can we pin this by checking out a git hash?

Reply:
Done. Why do we need this ?

Reply:
It's good to pin dependencies like these for more reproducibility and to avoid potential supply chain compromises.